content/docs/latest/installing/community-platforms/scaleway.md

```diff
@@ -0,0 +1,130 @@
+$ bzip2 --decompress ./flatcar_production_scaleway_image.img.bz2
+$ mv flatcar_production_scaleway_image.{img,qcow2}
+$ aws s3api create-bucket --bucket flatcar
+$ aws s3 cp flatcar_production_scaleway_image.qcow2 s3://flatcar/flatcar_production_scaleway_image.qcow2 
+$ scw instance snapshot create --wait \
+  zone=fr-par-1 name=flatcar-beta volume-type=l_ssd \
+  bucket=flatcar key=flatcar_production_scaleway_image.qcow2
+ID                1ead5e70-7e51-46bd-af3a-ca2d2fd9a8f2
+Name              flatcar-beta
+Organization      edca8b74-84f7-4a92-a8d6-0ea2eb816cb0
+Project           edca8b74-84f7-4a92-a8d6-0ea2eb816cb0
+VolumeType        l_ssd
+Size              8.5 GiB
+State             available
+CreationDate      2 minutes ago
+ModificationDate  1 second ago
+Zone              fr-par-1
+```
+
+## Butane Configs
+
+Flatcar allows you to configure machine parameters, launch systemd units on startup and more via Butane Configs. These configs are then transpiled into Ignition JSON configs and given to booting machines. Jump over to the [docs to learn about the supported features][butane-configs]. We're going to provide our Butane Config to Scaleway via the user-data flag. Our Butane Config will also contain SSH keys that will be used to connect to the instance.
+
+As an example, this Butane YAML config will start an Nginx Docker container and display the instance hostname:
+
+```yaml
+variant: flatcar
+version: 1.0.0
+passwd:
+  users:
+    - name: core
+      ssh_authorized_keys:
+        - ssh-rsa ABCD...
+storage:
+  directories:
+    - path: /var/www
+systemd:
+  units:
+    - name: nginx.service
+      enabled: true
+      contents: |
+        [Unit]
+        Description=NGINX example
+        After=docker.service coreos-metadata.service
+        Requires=docker.service coreos-metadata.service
+        [Service]
+        EnvironmentFile=/run/metadata/flatcar
+        TimeoutStartSec=0
+        ExecStartPre=-/usr/bin/docker rm --force nginx1
+        ExecStartPre=-/usr/bin/bash -c "echo 'Hello from ${COREOS_SCALEWAY_HOSTNAME}' > /var/www/index.html"
+        ExecStart=/usr/bin/docker run --name nginx1 --volume "/var/www:/usr/share/nginx/html:ro" --pull always --log-driver=journald --net host docker.io/nginx:1
+        ExecStop=/usr/bin/docker stop nginx1
+        Restart=always
+        RestartSec=5s
+        [Install]
+        WantedBy=multi-user.target
+```
+
+Transpile it to Ignition JSON:
+
+```bash
+cat butane.yaml | docker run --rm -i quay.io/coreos/butane:release > ignition.json
+```
+
+The `coreos-metadata.service` saves metadata variables to `/run/metadata/flatcar`. Systemd units can use them with `EnvironmentFile=/run/metadata/flatcar` in the `[Service]` section when setting `Requires=coreos-metadata.service` and `After=coreos-metadata.service` in the `[Unit]` section.
+
+## Launch machine
+
+Boot the machine with the CLI, referencing the snapshot ID from the import step above and your [Ignition file from Butane][butane-configs]:
```

Comment:
Maybe mention that the command will print the ID of the instance, which will be used in follow-up commands? Or add an example output of the `scw instance server create`?